### ISSUE TITLE
Handle time changes on the host

### ISSUE THREAD

**Issue by biddster** [biddster/node-red-contrib-schedex#11]
I've noticed that after a power cut, schedex starts and schedules on/off prior to the system clock being adjusted. Consequenly, the time skew is equal to the amount of time that schedex was turned off. 

Need to 'listen' for host clock changes and adapt our schedule.

**Comment by drmibell** [comment]
A related problem, for which I didn't want to raise a new issue, is that schedex seems not to update the times of suncalc events at the transitions between Standard Time and Daylight Saving Time (Summer Time) until after the first occurrence of each event on the new time standard. Perhaps schedex could call suncalc once a day (for example at 0300) to stay in sync.

**Comment by GoSpursGoNL** [comment]
Indeed, I noticed the same thing as @drmibell . Last night Standard Time began, but this night the lights will be activated one hour too late.

**Comment by tomSmartHome** [comment]
> I've noticed that after a power cut, schedex starts and schedules on/off prior to the system clock being corrected (presumably via ntp).
> 
> Consequently, the time skew is equal to the amount of time that schedex was turned off.
> 
> However, Schedex does recover once the first on/off triggers.
> 
> Need to 'listen' for host clock changes and adapt our schedule.

Any creative workarounds?

**Comment by biddster** [comment]
@drmibell @ GoSpursGoNL the daylight saving issue was fixed in 1.6.6.

@tomSmartHome - nothing springs to mind at the moment. Are you seeing the issue a lot?

**Comment by razvanstanga** [comment]
https://github.com/biddster/node-red-contrib-schedex/issues/93
